letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -326,12 +328,26 @@ def _find_best_vhost(self, target_name):
         # No winners here... is there only one reasonable vhost?
         if best_candidate is None:
             # reasonable == Not all _default_ addrs
-            reasonable_vhosts = self._non_default_vhosts()
+            # remove mod_macro hosts from reasonable vhosts
+            reasonable_vhosts = self._without_modmacro(
```

Comment:
Maybe just do this as a one-liner rather than defining a method? `_without_modmacro` is trivial, and you already have a comment here explaining what it does.